modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3019,8 +3019,15 @@ else if (valueString.isEmpty()) {
 			}
 		}
 		else if (sqlType == Types.DECIMAL) {
-			preparedStatement.setBigDecimal(
-				index, new BigDecimal(String.valueOf(value)));
+			String valueString = String.valueOf(value);
+
+			if (valueString.isEmpty()) {
+				preparedStatement.setBigDecimal(index, new BigDecimal(0L));
+			}
+			else {
+				preparedStatement.setBigDecimal(
+					index, new BigDecimal(valueString));
+			}
```

Comment:
@Valmir-unicap this issue is also fixed by this another [PR](https://github.com/brianchandotcom/liferay-portal/pull/131345/files) which was already merged.

Therefore, I'll close this PR and move the ticket to resolved.

Thank you for your work on this.